src/OctoshiftCLI.Tests/Octoshift/Services/GithubApiTests.cs

```diff
@@ -3368,12 +3368,52 @@ public async Task AbortMigration_Returns_True_On_Success()
             .ReturnsAsync(response);
 
         // Act
-        var expectedBooleanResponse = await _githubApi.AbortMigration(migrationSourceId);
+        var expectedBooleanResponse = await _githubApi.AbortMigration(migrationId);
 
         // Assert
         expectedBooleanResponse.Should().Be(actualBooleanResponse);
     }
 
+    [Fact]
+    public async Task AbortMigration_Surfaces_Error_With_Incorrect_Migration_ID()
+    {
+        // Arrange
+        const string migrationId = "1234";
+
+        const string query = @"
+                mutation abortRepositoryMigration(
+                    $migrationId: ID!,
+                )";
+        const string gql = @"
+                abortRepositoryMigration(
+                    input: { 
+                        migrationId: $migrationId
+                    })
+                   { success }";
+
+        var payload = new
+        {
+            query = $"{query} {{ {gql} }}",
+            variables = new
+            {
+                migrationId,
+            },
+            operationName = "abortRepositoryMigration"
+        };
```

Comment:
there is no actual setup code here and this test will fail because of it. You need to setup the `_githubClientMock` to throw a `OctoshiftCliException` with the desired message:

```c#
_githubClientMock
    .Setup(m => m.PostGraphQLAsync(It.IsAny<string>(), It.IsAny<object>(), null))
    .ThrowsAsync(new OctoshiftCliException("Could not resolve to a node"));
```